src/dask_awkward/lib/core.py

```diff
@@ -532,6 +530,11 @@ def f(self, other):
                     meta = op(other, self._meta)
                 else:
                     meta = op(self._meta, other)
+            if meta.ndim:
```

Comment:
This applies to the scenario, which apparently we hadn't anticipated, that an operation on a scalar produces an array.